pkg/clients/ec2/vpccidrblock.go

```diff
@@ -39,16 +39,8 @@ func (r *CIDRNotFoundError) Error() string {
 
 // IsCIDRNotFound returns true if the error code indicates that the CIDR Block Association was not found
 func IsCIDRNotFound(err error) bool {
-	if _, ok := err.(*CIDRNotFoundError); ok {
```

Comment:
should this still look for both "cidr not found" and "cidr association not found", or is that one error? cc @enderv